charts/fluid/fluid/values.yaml

```diff
@@ -149,6 +149,8 @@ webhook:
   reinvocationPolicy: IfNeeded
   tolerations:
     - operator: Exists
+  # if configmap `tiered-locality-config` exists and not want to replace the content, set this to false.
+  updateTieredLocality: true
```

Comment:
How about changing `updateTieredLocality` to `forceReplaceTieredLocality`? And by default, the value is false. Thanks.